### PR TITLE
ISSUE-72: zkclient throws an UnknownHostException if one of the servers in zkConnectionString doesn't resolve.

### PR DESCRIPTION
**Pull request by harshach**

### CODE REVIEW

**Comment by harshach** [comment]
@jzillmann when you get a chance can you review this.

File: src/main/java/org/I0Itec/zkclient/ZkConnection.java [hunk 1]
```diff
@@ -66,10 +73,12 @@ public void connect(Watcher watcher) {
                 throw new IllegalStateException("zk client has already been started");
             }
             try {
-                LOG.debug("Creating new ZookKeeper instance to connect to " + _servers + ".");
-                _zk = new ZooKeeper(_servers, _sessionTimeOut, watcher);
+                LOG.info("Resolving hosts from configured hosts " + _servers);
```
Comment (jzillmann) [inline src/main/java/org/I0Itec/zkclient/ZkConnection.java]: Not sure if those should be debug or info messages...

File: src/test/java/org/I0Itec/zkclient/ZkConnectionTest.java [hunk 2]
```diff
@@ -27,6 +27,7 @@ public ZkConnectionTest() {
         super(establishConnection());
     }
 
+
```
Comment (jzillmann) [inline src/test/java/org/I0Itec/zkclient/ZkConnectionTest.java]: Unnecessary line..

File: src/test/java/org/I0Itec/zkclient/ZkStateChangeTest.java [hunk 3]
```diff
@@ -203,6 +203,12 @@ public String getServers() {
             return "test";
         }
 
+        @Override
+        public String getResolvedServers() {
+            return "test";
+        }
+
+
```
Comment (jzillmann) [inline src/test/java/org/I0Itec/zkclient/ZkStateChangeTest.java]: Unnecessary line..

**Comment by harshach** [comment]
Thanks for the review @jzillmann . Updated the PR with the changes.

**Comment by harshach** [comment]
@jzillmann yes once zookeeper releases 3.5 and we upgrade the client version we can revert this.

**Comment by jzillmann** [comment]
Thanks @harshach merged it!
Let me know if you need a release!

**Pull request merged**